lld/test/ELF/avr-errors.s

```diff
@@ -0,0 +1,33 @@
+; REQUIRES: avr
```

Comment:
avr-reloc-error.s will be a better name to be similar to x86-64-reloc-error.s.

Reply:
Thanks for your review. I will fix all your concerns when commit my patch.